engine/userguide/eng-image/dockerfile_best-practices.md

```diff
@@ -283,6 +283,19 @@ keep the image size down. Since the `RUN` statement starts with
 > **Note**: The official Debian and Ubuntu images [automatically run `apt-get clean`](https://github.com/docker/docker/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105),
 > so explicit invocation is not required.
 
+#### Using pipes
+
+Where your `RUN` command depends on pipes, for example:
```

Comment:
Not all of our users are UNIX experts. Let's give them some context, and also not start with a sentence fragment:

Some `RUN` commands depend on the ability to pipe the output of one command into another, using the pipe character (`|`), as in the following example: